### DISCUSSION TITLE
Client facing runtime exception

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#2510] [General]
My client getting runtime exception. What it is ? Can we handle any exception error in AS3. Client Image error is as follow : 

![WhatsApp Image 2023-03-03 at 10 25 43 AM](https://user-images.githubusercontent.com/37571867/222743608-8a1ffcc5-71a7-4052-8186-0117ab239e4b.jpeg)

@ajwfrost please help me?

**Comment by ajwfrost** [comment]
That looks like 'app not responding', but while the app is in the background..? If you're able to set up developer mode on the phone and pull out the crash report, we can try to look at the call stack to see what's going on.

Have you looked at the "run in background thread" option? This may help avoid these sorts of things:
https://airsdk.dev/docs/building/application-descriptor-files/elements/android#runtimeinbackgroundthread

thanks
